tsconfig.json

```diff
@@ -0,0 +1,62 @@
+{
+  "include": [
+    "./src/*.ts",
```

Comment:
you don't need this line if you have the next

Reply:
Yeah I dropped the include bit altogether now actually.